R/teal_data_module.R

```diff
@@ -144,10 +143,17 @@ teal_data_module <- function(ui, server, label = "data module", once = TRUE) {
 teal_transform_module <- function(ui = function(id) NULL,
                                   server = function(id, data) data,
                                   label = "transform module",
-                                  datanames = "all") {
+                                  datanames = character(0)) {
   checkmate::assert_function(ui, args = "id", nargs = 1)
   checkmate::assert_function(server, args = c("id", "data"), nargs = 2)
   checkmate::assert_string(label)
+  checkmate::assert_character(datanames)
+  if (identical(datanames, "all")) {
+    stop(
+      "teal_transform_module can't have datanames property equal to 'all'. Set `datanames = character(0)` instead.",
+      call. = FALSE
+    )
+  }
```

Comment:
```suggestion
  stopifnot(
    "teal_transform_module can't have datanames property equal to 'all'. Set `datanames = character(0)` instead." = 
      !identical(datanames, "all")
  )
```

Reply:
Nevermind about this. It creates a problem with glue formatter. 

The code is very readable

Reply:
Ignoring ;)